tools/flaky_test_monitor/testdata/summary1/expected/test-result-crypto-hash-1-count-fail.json

```diff
@@ -1,124 +0,0 @@
-{
```

Comment:
why are all the test data files deleted? They're required for all the unit tests for level 1, 2, 3